docs/tempo/website/configuration/compression.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Compression/Encoding
+title: Compression and encoding
 weight: 5
 ---
 
-# Compression/Encoding
+# Compression and encoding
 
-Tempo has the ability to compress traces that it pushes into the backend. This requires a bit extra
-memory and cpu but seriously reduces the amount of stored data.  Anecdotal tests suggest that zstd will
+Tempo can compress traces that it pushes into the backend. This requires a bit extra
+memory and CPU but reduces the amount of stored data.  Anecdotal tests suggest that `zstd` will
```

Comment:
I feel this wording is unpolished.  Here's a suggestion.  Not a necessary change.
```suggestion
Tempo can compress traces that it pushes to backend storage. This requires extra
memory and CPU, but it reduces the quantity of stored data.  Anecdotal tests suggest that `zstd` will
```